subsys/testsuite/include/negative_handler.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr.h>
+
+
+#if defined(CONFIG_TEST_NEGATIVE_FATAL_HANDLER)
+/* These valid flag used to figure out if the fault was expected or not. */
+extern volatile bool valid_fault;
+
+/**
+ * @brief Set the flag that treat fatal error happened as expected
+ *
+ * This is used for negative test cases which will trigger a fatal error.
+ * Set the param true will still pass the test case when expected fatal
+ * error happened. For normal test case, set it false will make it back to
+ * normal situation.
+ *
+ * @param valid flag indicate fault is expected
+ */
+static inline void set_fault_valid(bool valid)
+{
+	valid_fault = valid;
+	/* Put a barrier here, such that no instructions get ordered by the
+	 * compiler before we set valid_fault. This can happen with expansion
+	 * of inline syscall invocation functions.
+	 */
+	compiler_barrier();
+}
+#endif
+
+
+#if defined(CONFIG_TEST_NEGATIVE_ASSERT_HANDLER)
+/* These valid flag used to figure out if the assert  was expected or not. */
+extern volatile bool valid_assert;
+
+/**
+ * @brief Set the flag that treat assert fail happened as expected
+ *
+ * This is used for negative test cases which will trigger a assert fail.
```

Comment:
Please remove "will", and replace "trigger" with "triggers"

Reply:
Thanks for correcting this. Got it fixed.